### PR TITLE
[Private] Rename some models

### PR DESCRIPTION
**Pull request by dvbb**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.0.1 (2022-12-01)
```
Comment (Yao725) [inline sdk/dns/Azure.ResourceManager.Dns/CHANGELOG.md:3]: Actually I think we haven't release the stable version for Dns, could you please double check and update the changelog?

File: sdk/dns/Azure.ResourceManager.Dns/src/Azure.ResourceManager.Dns.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.0.0</Version>
+    <Version>1.0.1</Version>
```
Comment (Yao725) [inline sdk/dns/Azure.ResourceManager.Dns/src/Azure.ResourceManager.Dns.csproj:3]: Please double check the version

File: sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs [hunk 3]
```diff
@@ -26,7 +26,7 @@ public partial class PrivateDnsAaaaRecordResource : Azure.ResourceManager.ArmRes
         protected PrivateDnsAaaaRecordResource() { }
         public virtual Azure.ResourceManager.PrivateDns.PrivateDnsAaaaRecordData Data { get { throw null; } }
         public virtual bool HasData { get { throw null; } }
-        public static Azure.Core.ResourceIdentifier CreateResourceIdentifier(string subscriptionId, string resourceGroupName, string privateZoneName, string aaaaRecordName) { throw null; }
+        public static Azure.Core.ResourceIdentifier CreateResourceIdentifier(string subscriptionId, string resourceGroupName, string PrivateDnsZoneName, string aaaaRecordName) { throw null; }
```
Comment (Yao725) [inline sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs:29]: I think here we should use `privateDnsZoneName`, please double check all the related methods.

File: sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs [hunk 4]
```diff
@@ -301,32 +301,32 @@ public partial class PrivateDnsTxtRecordResource : Azure.ResourceManager.ArmReso
...
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.PrivateDns.PrivateDnsTxtRecordResource>> UpdateAsync(Azure.ResourceManager.PrivateDns.PrivateDnsTxtRecordData data, Azure.ETag? ifMatch = default(Azure.ETag?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class PrivateZoneCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.PrivateDns.PrivateZoneResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.PrivateDns.PrivateZoneResource>, System.Collections.IEnumerable
+    public partial class PrivateDnsZoneCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.PrivateDns.PrivateDnsZoneResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.PrivateDns.PrivateDnsZoneResource>, System.Collections.IEnumerable
     {
```
Comment (Yao725) [inline sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs:313]: Should we also rename the parameter name from `privateZoneName` to `privateDnsZoneName`?

Comment (dvbb) [reply sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs:313]: Reversed. Now all of the parameters are the same with Generated code, naming `privateZoneName`

**Review by Yao725 — APPROVED** [review]

**Pull request merged**